src/App.js

```diff
@@ -1,26 +1,38 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import style from './App.css';
+import { Provider} from 'react-redux';
+import {createStore, applyMiddleware, compose } from 'redux';
+import { restaurantsReducer } from './reducers/Reducers.js';
+import bkgImage  from './restaurant_image.jpg';
+import RestaurantsList from './containers/RestaurantsList.js'
+import thunk from 'redux-thunk';
+
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+const store = createStore(restaurantsReducer, {
+  isFetching: false,
+  items: [],
+  error: undefined,
+  lastUpdate: undefined,
+},
+  composeEnhancers(applyMiddleware(thunk))
+);
 
 function App() {
   return (
-    <div className="App">
+      <Provider store={store}>
+    <div className={style.App}>
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <img src={ bkgImage }
```

Comment:
minor/sidenote:
we try to use full meaningful names.
bkgImage -> backgroundImage

in this case it was clear, but that is not always the case.